packages/hash/api/src/graphql/resolvers/knowledge/block/block.ts

```diff
@@ -1,16 +1,19 @@
 import { getBlockById } from "../../../../graph/knowledge/system-types/block";
 import { QueryBlocksArgs, ResolverFn } from "../../../api-types.gen";
 import { GraphQLContext } from "../../../context";
+import { dataSourceToImpureGraphContext } from "../../util";
 import { UnresolvedBlockGQL } from "../graphql-mapping";
 
 export const blocksResolver: ResolverFn<
   Promise<UnresolvedBlockGQL[]>,
   {},
   GraphQLContext,
   QueryBlocksArgs
-> = async (_, params, { dataSources: { graphApi } }) => {
+> = async (_, params, { dataSources }) => {
+  const ctx = dataSourceToImpureGraphContext(dataSources);
```

Comment:
Nit: can we try and use `context` instead of `ctx`? It's more consistent with the rest of our code.

Reply:
Sorry, i actively wanted this to be `context`, but I seem to have missed these. Thanks!

Reply:
eccdaa1a01bc30a69de4100d385ad13ee319892c